ui/src/main/webapp/app/query/view-builder/view-builder.html

```diff
@@ -8,42 +8,28 @@ <h3 class="md-title section-title">Add Filters</h3>
     <div ng-show="ctrl.showBuilder">
         <div layout="row" layout-padding>
             <md-input-container flex>
-                <label>Only include these entities</label>
-                <md-select id="view-entities" class="no-ellipsis" multiple
-                           md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entities')"
-                           ng-model="ctrl.model.viewEntities"
-                           ng-change="ctrl.onElementGroupChange('entity')"
-                           data-md-container-class="searchable-select">
+                <label>Only include these entity types</label>
+                <md-select id="view-entities" class="no-ellipsis" multiple md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entity types')"
```

Comment:
I think this needs to stay as entities

Reply:
The <label>entity types or the (ctrl.model.viewEntities, 'entity types')?